examples/grpc/greeter/Makefile

```diff
@@ -0,0 +1,14 @@
+CXXFLAGS ?= -I../../../include
```

Comment:
Since we want this to be portable, eventually this should become a CMakefile.txt.. but we can leave it for now if you want.